Lib/fontTools/colorLib/builder.py

```diff
@@ -107,20 +200,18 @@ def populateCOLRv0(
 
 def buildCOLR(
     colorGlyphs: _ColorGlyphsDict,
-    version: Optional[int] = None,
+    version: int = 0,
```

Comment:
I wanted to avoid trying to guess from weak types (a dict with some tuples in it? - clearly you sir want v1!) what format was intended and to therefore be able to adjust the v1 input format without having to worry I'd made it something that could be mistaken for a v0 input.

UFO storing COLR version makes sense to me. Is that undesirable?

It's plausible to check for v1 glyphs that could be made into v0 ones at the end of building a v1 if we want?

Reply:
Having thought about it a bit more I think it's not as bad as I thought. A 2-tuple (str, int) isn't really a useful way to define a v1 paint. I think I can put this back.